packages/ui-components/src/components/absolute-time-picker/absolute-time-picker.scss

```diff
@@ -16,7 +16,8 @@
 		width: fit-content;
 
 		kv-icon {
-			--icon-color: #{kv-color('neutral-4')};}
+			--icon-color: #{kv-color('neutral-4')};
```

Comment:
I believe this was introduced by the linter in the fix process but in terms of readability, I updated because it made more sense having the closing bracket bellow the kv-icon tag. I can rever it though but personally I think this change made more sense since we are following this approach everywhere.

Reply:
yes, I miss that part :(